src/apps/backend/modules/comment/types.ts

```diff
@@ -0,0 +1,29 @@
+import { ApplicationError } from '../application';
+import { HttpStatusCodes } from '../http';
+
+export class Comment {
+  id: string;
+  account: string;
+  taskId: string;
+  content: string;
+}
+
+export type CreateCommentParams = {
+  accountId: string;
+  content: string;
+  taskId: string;
+};
+
+export enum CommentErrorCode {
+  NOT_FOUND = 'COMMENT_ERR_01',
+}
+
+export class CommentNotFoundError extends ApplicationError {
+  code: CommentErrorCode;
+
+  constructor(taskId: string) {
+    super(`Comment with taskId ${taskId} not found.`);
```

Comment:
Error should offer solution instead of blaming or just surfacing the problem. Please google "how to write good error messages"